test/test/cpu/CPUTimeService.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright The async-profiler authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package test.cpu;
+
+import com.sun.tools.attach.AttachNotSupportedException;
+import com.sun.tools.attach.VirtualMachine;
+
+import javax.management.MBeanServerConnection;
+import javax.management.ObjectName;
+import javax.management.remote.JMXConnector;
+import javax.management.remote.JMXConnectorFactory;
+import javax.management.remote.JMXServiceURL;
+import java.io.IOException;
+import java.util.Set;
+
+/**
+ * Obtain the CPU time of a process using the Attach API
+ */
+class CPUTimeService implements AutoCloseable {
```

Comment:
`CpuTimeService` - for the consistency with other class names.
The same for the method name. Note that MBean attribute is also called `ProcessCpuTime`.